File: BasicCourseOOP/1. CursoPOOUber/PHP/Account.php
<?php
    class Account{
        public $id=integer;
        public $name=string;
        public $document=string;
        public $email=string;
        public $password=string;

        public function __construct($name, $document){
            $this->name=$name;
            $this->document=$document;
        }
    }

File: BasicCourseOOP/1. CursoPOOUber/Python/account.py
from pydoc import doc
import string


class Account:
    id=int
    name=str
    document=str
    email=str
    password=str

    # def __init__(self,name,document):
    #     self.name=name
    #     self.document=document

    def __init__(self,name,document,email,password):
        self.name=name
        self.document=document
        self.email=email
        self.password=password